packages/website/ts/pages/governance/governance.tsx

```diff
@@ -0,0 +1,299 @@
+import { BigNumber } from '@0x/utils';
+import * as _ from 'lodash';
+import * as React from 'react';
+import DocumentTitle from 'react-document-title';
+import styled from 'styled-components';
+import { Banner } from 'ts/components/banner';
+import { Button } from 'ts/components/button';
+import { ModalContact } from 'ts/components/modals/modal_contact';
+import { Column, FlexWrap, Section } from 'ts/components/newLayout';
+import { SiteWrap } from 'ts/components/siteWrap';
+import { Heading, Paragraph } from 'ts/components/text';
+import { Countdown } from 'ts/pages/governance/countdown';
+import { ModalVote } from 'ts/pages/governance/modal_vote';
+import { RatingBar } from 'ts/pages/governance/rating_bar';
+import { VoteInfo, VoteValue } from 'ts/pages/governance/vote_form';
+import { VoteStats } from 'ts/pages/governance/vote_stats';
+import { colors } from 'ts/style/colors';
+import { configs } from 'ts/utils/configs';
+import { utils } from 'ts/utils/utils';
+
+interface LabelInterface {
+    [key: number]: string;
+}
+
+export interface TallyInterface {
+    zeip?: string;
+    yes?: BigNumber;
+    no?: BigNumber;
+    blockNumber?: string;
+    totalVotes?: string;
+    totalBalance?: BigNumber;
+}
+
+interface State {
+    isContactModalOpen: boolean;
+    isVoteModalOpen: boolean;
+    isWalletConnected: boolean;
+    isVoteReceived: boolean;
+    providerName?: string;
+    tally?: TallyInterface;
+}
+
+const benefitLabels: LabelInterface = {
+    1: 'Little Benefit',
+    2: 'Medium Benefit',
+    3: 'High Benefit',
+};
+
+const riskLabels: LabelInterface = {
+    1: 'Low Risk',
+    2: 'Medium Risk',
+    3: 'High Risk',
+};
+
+const proposalData = {
+    zeipId: 23,
+    title: 'ZEIP23: Trading Bundles of Assets',
+    summary: `This ZEIP introduces the MultiAssetProxy which brings support for trading arbitrary bundles of assets in 0x protocol. Historically only a single asset could be traded per each side of a trade. With the introduction of the MultiAssetProxy, users will be able to trade multiple ERC721 assets and even mix ERC721 and ERC20 to trade in a single order.`,
+    url: 'https://blog.0xproject.com/zeip-23-trade-bundles-of-assets-fe69eb3ed960',
+    votingDeadline: 1551584000,
+    benefit: {
+        title: 'Benefit',
+        summary: `Supporting bundled trades is one of the most commonly requested features since the launch of 0x v2. Demand originated from our discussions with gaming and NFT related projects, but this upgrade provides utility to Prediction Markets as well as any relayer that wants to offer baskets of tokens. The MultiAssetProxy will enable new possibilities of trading.`,
```

Comment:
`Supporting trades for bundles of assets has been one of the most commonly requested features since the launch of 0x v2. The idea for this feature originated from discussions with gaming and NFT related projects. However, this upgrade also provides utility to relayers for prediction markets or baskets of tokens. The MultiAssetProxy will enable brand new ways of trading.`